pyaedt/modeler/Object3d.py

```diff
@@ -1,13 +1,6 @@
-# -*- coding: utf-8 -*-
```

Comment:
Please add it back

Reply:
Didn't realize this, thanks!  We should verify that our other source files have this, we might have removed this in others.

Reply:
@maxcapodi78 If you want me to put this line in all the PY files, let me know. If it only needs to go in some of the files, provide a list and I'll add it only to those files.

Reply:
@PipKat for the moment should be fine. Ideally it has to be only in files with special characters that Ipy is not able to understand if utf-8 is not explicitily written. once we will have Ironpython UnitTest in place will be easy to detect issues on this topic.